### ISSUE TITLE
Ungrouped processes index filters not working

### ISSUE THREAD

**Issue by verarojman** [Platoniq/decidim-catencomu#78]
## 🔁 How to reproduce

1. Create 2 published and ungrouped participatory processes:

```ruby
Decidim::ParticipatoryProcess.create!([
  {
    decidim_organization_id: 1, 
    slug: "ungrouped-process-1",
    title: I18n.available_locales.map {|l| [l, "Ungrouped Process 1"] }.to_h,
    subtitle: I18n.available_locales.map {|l| [l, "Ungrouped Process 1"] }.to_h,
    description: I18n.available_locales.map {|l| [l, "Ungrouped Process 1"] }.to_h,
    short_description: I18n.available_locales.map {|l| [l, "Ungrouped Process 1"] }.to_h,
    published_at: Time.now,
    start_date: 1.day.ago,
    end_date: 1.month.from_now
  },
  {
    decidim_organization_id: 1,
    slug: "ungrouped-process-2",
    title: I18n.available_locales.map {|l| [l, "Ungrouped Process 2"] }.to_h,
    subtitle: I18n.available_locales.map {|l| [l, "Ungrouped Process 2"] }.to_h,
    description: I18n.available_locales.map {|l| [l, "Ungrouped Process 2"] }.to_h,
    short_description: I18n.available_locales.map {|l| [l, "Ungrouped Process 2"] }.to_h,
    published_at: Time.now,
    start_date: 1.month.ago,
    end_date: 1.day.ago
  }
])
```

2. Visit http://localhost:3000/global_processes

3. Click on either of the tabs to toggle processes (Current / Past / All)
![image](https://user-images.githubusercontent.com/8806781/127737768-b35e0388-63fc-41dd-9dfa-24b453e78994.png)

⚠️  A JS request is triggered but it's being intercepted by ParticipatoryProcessScoper and never gets to reload the page with the filtered participatory process results. This is what we have to fix, since it's blocking specs from `spec/system/participatory_processes_spec.rb` in https://github.com/Platoniq/decidim-catencomu/pull/74

**Comment by aitorlb** [comment]
I can reproduce the issue in `feature/main-processes-index-override` branch, but not in `main`.

**Comment by verarojman** [comment]
Fixed by #79
